Exercises_2023.md

```diff
@@ -0,0 +1,70 @@
+
++ In two big teams, studens have to fork a python project that needs troubleshooting, then fix it in different stages taking charge of coordination and task distribution.
++ Every student forks the final project from their team, runs it in their PC, and they submit the individual answer under the corresponding issue on the course repo.
++ To prepare for week 3, teams will be assign a big scientific collaboration to investigate and report about in week 3.
+
+## Week 3: big science, scale vs precision vs control
+*keywords:* Big-science collaborations, Citizen science, Credit and recognition, Policy and politics of science, Governance structures, Independence and thriving, Relativism of big, Climate impacts and transformation of the research ecosystem
+
++ Students investigate the structure, history, function, and working conditions of these big collaborations and present for the class. Presentations will recieve peer-feedback.
+	+ IPCC, Roderick van der Wal, Aimee Slangen,
+	+ ATLAS, Alessandro Grelli, 
+	+ LIGO/VIRGO, Chris van den Broeke
+	+ NEEM, ?? 
+	+ Spectroscopy, Frans Snik
+	+ Astroplant https://www.astroplant.io/, (?)
+	+ Pangeo, Ryan Abernati
+	+ SKA, 
+	+ Chinese space agency,
+	+ ECMWF (KNMI), 
+	+ Seti at home,
+	+ Large scale environment sensing, Rolf Hut
+	 
++ How is open science implement in the structure of the big collaborations, and does it help?
+
+## Week 4: documentation for reproducibility
+*keywords:* Best practices for documentation, Data structures and data plans, GDPR and privacy rules, templates and bug reports, Persisten identifiers, Machine-readable docs, Systematic review, Scripting, Compatibility, Dockers-Virtual Environments-Pinning, Best practices in data visualization, 
+
++ Student teams start with their measurement project, define the goal and the processes, and create the documentation folder by following the lecture tips.
+
+
+## Week 5: open design and open hardware; prototyping
+*keywords:* Scientific design environments, Open protocols, Microcontrollers, Human-computer interface, Automation and bots, Remote labs, Materials-resources-constraints, Defense and national security, Maintenance, Prototyping
+
++ Each team presents their project to the class
++ Teams have to reproduce each-others results and write a review on 
+
+## Week 6: public engagement: story-telling and presentation; activism and trust in scientists
+*keywords:* How storytelling works, Science communication and journalism, Activism and trust in Science, Bounds of expertise, Academic social contract, Metaphores, Communicating uncertainty, Narratives and framing, Difficult to reach audience, Process and outcome
+
++ lay person summary of a previous or future project
++ Identify jargon and discuss how to avoid it in the science communication (inspiration from upgoer 5)
```

Comment:
```suggestion
+ In pairs, the students identify the jargon in their layperson summaries, and discuss how to avoid it in the science communication (inspiration from upgoer 5)
```